### DISCUSSION TITLE
Question: do the 'private let readMe=""" ... """ ' blocks get bundled with an app?

### DISCUSSION THREAD

**Discussion by andrewjroberts** [pointfreeco/swift-composable-architecture#1569] [Q&A] [answered]
Hi all,

Quick question on some iOS/Swift fundamentals related to something in this repo:

Would the `private let readMe=""" ... """` blocks in the CaseStudies examples get bundled with an app if they were not cleaned up in some build step?

I really like how clear the developer experience is of reading these blocks, and I've been using them to include links to API references, Apple documentation, etc.—but I'm wondering if I need to write a cleanup script before archiving for a build.

**Comment by tgrapperon** [comment, marked as answer]
Hey @andwrobs. It this case, they are bundled because they are referenced/used in the `View` itself to create the `AboutView`.
If it wasn't the case, they would likely be pruned by Swift during the optimization phase.

If you want to add API reference, documentation and links, the best solution is probably to harness DocC annotations though. It was made exactly for this purpose.

**Reply by andrewjroberts** [reply]
@tgrapperon thank you for both answering & pointing me in a better direction :-) cheers
